### PR TITLE
WebView2: Fixed broken link and added missing member references

### PR DESCRIPTION
**Pull request by peiche-jessica**

### CODE REVIEW

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md [hunk 1]
```diff
@@ -23,16 +23,17 @@ This represents the WebView2 Environment.
 --------------------------------|---------------------------------------------
 [BrowserVersionString](#browserversionstring) | The browser version info of the current CoreWebView2Environment, including channel name if it is not the stable channel.
 [NewBrowserVersionAvailable](#newbrowserversionavailable) | NewBrowserVersionAvailable fires when a newer version of the Edge browser is installed and available for use via WebView2.
+[CompareBrowserVersions](#comparebrowserversions) | Compare browser versions correctly to determine which version is newer, older or same.
```
Comment (Reezaali) [inline microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:26]:
```suggestion
[CompareBrowserVersions](#comparebrowserversions) | Compare browser versions to determine if they match or are different.
```

Comment (Reezaali) [reply microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:26]: Can we clarify what 2+ things are being compared? For example, are we comparing the installed version of Microsoft Edge and the Runtime?

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md [hunk 2]
```diff
@@ -23,16 +23,17 @@ This represents the WebView2 Environment.
...
+[CreateAsync](#createasync) | Creates an evergreen WebView2 Environment using the installed Edge version.
```
Comment (Reezaali) [inline microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:27]:
```suggestion
[CreateAsync](#createasync) | Creates an evergreen WebView2 Environment using the installed version of Microsoft Edge.
```

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md [hunk 3]
```diff
@@ -23,16 +23,17 @@ This represents the WebView2 Environment.
...
 [CreateCoreWebView2CompositionControllerAsync](#createcorewebview2compositioncontrollerasync) | Asynchronously create a new WebView for use with visual hosting.
 [CreateCoreWebView2ControllerAsync](#createcorewebview2controllerasync) | Asynchronously create a new WebView.
 [CreateCoreWebView2PointerInfo](#createcorewebview2pointerinfo) | Create an empty CoreWebView2PointerInfo.
 [CreateWebResourceResponse](#createwebresourceresponse) | Create a new web resource response object.
+[GetAvailableBrowserVersionString](#getavailablebrowserversionstring) | Get the browser version info including channel name if it is not the stable channel or the Embedded Edge.
```
Comment (Reezaali) [inline microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:32]:
```suggestion
[GetAvailableBrowserVersionString](#getavailablebrowserversionstring) | Get the browser version information. You also get the channel name if the channel is not a stable channel, or you are using the embedded version of Microsoft Edge.
```

Comment (Reezaali) [reply microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:32]: Does "embedded" mean the mobile version of Microsoft Edge?

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md [hunk 4]
```diff
@@ -53,6 +54,32 @@ To use the newer version of the browser you must create a new environment and We
 
 Because a user data folder can only be used by one browser process at a time, if you want to use the same user data folder in the WebViews using the new version of the browser, you must close the environment and WebViews that are using the older version of the browser first. Or simply prompt the user to restart the app.
 
+#### CompareBrowserVersions 
+
+Compare browser versions correctly to determine which version is newer, older or same.
```
Comment (Reezaali) [inline microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:59]: Same comment as above.

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md [hunk 5]
```diff
@@ -53,6 +54,32 @@ To use the newer version of the browser you must create a new environment and We
...
+
+> public static int [CompareBrowserVersions](#comparebrowserversions)(string version1, string version2)
+
+Returns -1, 0 or 1 depending on whether version1 is less than, equal to or greater than version2, respectively.
```
Comment (Reezaali) [inline microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:63]:
```suggestion
Returns -1, 0 or 1 depending on whether the first version is less than, equal to or greater than the second version being compared.
```

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md [hunk 6]
```diff
@@ -53,6 +54,32 @@ To use the newer version of the browser you must create a new environment and We
...
+
+##### Parameters
+* `version1` One of the version strings to compare.
```
Comment (Reezaali) [inline microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:66]:
```suggestion
* `version1` The first version to compare. 
```

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md [hunk 7]
```diff
@@ -53,6 +54,32 @@ To use the newer version of the browser you must create a new environment and We
...
+Compare browser versions correctly to determine which version is newer, older or same.
+
+> public static int [CompareBrowserVersions](#comparebrowserversions)(string version1, string version2)
+
+Returns -1, 0 or 1 depending on whether version1 is less than, equal to or greater than version2, respectively.
+
+##### Parameters
+* `version1` One of the version strings to compare. 
+
+* `version2` The other version string to compare.
```
Comment (Reezaali) [inline microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:68]:
```suggestion
* `version2` The second version to compare.
```

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md [hunk 8]
```diff
@@ -53,6 +54,32 @@ To use the newer version of the browser you must create a new environment and We
...
+
+#### CreateAsync 
+
+Creates an evergreen WebView2 Environment using the installed Edge version.
```
Comment (Reezaali) [inline microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:72]: Same comment as above.

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md [hunk 9]
```diff
@@ -53,6 +54,32 @@ To use the newer version of the browser you must create a new environment and We
...
+
+> public static async Task< [CoreWebView2Environment](microsoft-web-webview2-core-corewebview2environment.md) > [CreateAsync](#createasync)(string browserExecutableFolder, string userDataFolder, CoreWebView2EnvironmentOptions options)
+
+##### Parameters
+* `browserExecutableFolder` The relative path to the folder that contains the embedded Edge.
```
Comment (Reezaali) [inline microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:77]: embedded Edge - similar comment to above. Can we clarify?

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md [hunk 10]
```diff
@@ -95,6 +122,15 @@ Create a new web resource response object.
 
 The headers is the raw response header string delimited by newline. It's also possible to create this object with empty headers string and then use the CoreWebView2HttpResponseHeaders to construct the headers line by line. For information on other parameters see CoreWebView2WebResourceResponse.
 
+#### GetAvailableBrowserVersionString 
+
+Get the browser version info including channel name if it is not the stable channel or the Embedded Edge.
```
Comment (Reezaali) [inline microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:127]: Same comment as above.

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md [hunk 11]
```diff
@@ -95,6 +122,15 @@ Create a new web resource response object.
...
+
+> public static string [GetAvailableBrowserVersionString](#getavailablebrowserversionstring)(string browserExecutableFolder)
+
+##### Parameters
+* `browserExecutableFolder` The relative path to the folder that contains the embedded Edge.
```
Comment (Reezaali) [inline microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:132]: Same comment as above.

**Review by Reezaali — CHANGES_REQUESTED** [review]

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md (hunk 5 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:63]: I believe the version1 and version2 here refer to the parameter names in the API signature.

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md (hunk 3 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:32]: The 'Embedded Edge' here should be 'WebView2 Runtime'. There is no channel name for WebView2 Runtime, either.

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md (hunk 1 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:26]: Any browser version string. One can compare 86.0.622.11 to 87.0.628.0, for instance.

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md (hunk 3 above)
Comment (Reezaali) [reply microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:32]: [GetAvailableBrowserVersionString](#getavailablebrowserversionstring) | Get the browser version information. You also get the channel name if the channel is not a stable channel.

Does that work?

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md (hunk 5 above)
Comment (Reezaali) [reply microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:63]: Yeah, I think spelling it out (the first version) helps with clarity.

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md (hunk 9 above)
Comment (peiche-jessica) [inline microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:77]:
```suggestion
* `browserExecutableFolder` The relative path to the folder that contains the fixed version of the WebView2 Runtime. 
```

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md (hunk 11 above)
Comment (peiche-jessica) [inline microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:132]:
```suggestion
* `browserExecutableFolder` The relative path to the folder that contains the fixed version of the WebView2 Runtime.
```

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md (hunk 1 above)
Comment (Reezaali) [reply microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:26]: Thanks for clarifying. Do we have scenarios when customers would need to use this method? Or perhaps would it be used with some other APIs? That'll help clarify when customers can use this method and in what situation.

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md (hunk 3 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:32]: Can we also mention that end developers won't see channel name if they are using WebView2 Runtime?

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md (hunk 1 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:26]: For example, they could take the result from calling GetAvailableCoreWebView2BrowserVersionString and compare it with the SDK version they want to use. [This](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/0-9-622/webview2-idl#comparebrowserversions) is the win32 version for more details.

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md (hunk 3 above)
Comment (Reezaali) [reply microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:32]: GetAvailableBrowserVersionString | Get the browser version information. You also get the channel name if the channel is not a stable channel. If you use the WebView Runtime, no channel name returned.

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md (hunk 1 above)
Comment (Reezaali) [reply microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:26]: I like that clarification. Can we add that?

File: microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md (hunk 8 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md:74]:
```suggestion
Creates an evergreen WebView2 Environment using the installed version of Microsoft Edge.
```

**Review by Reezaali — APPROVED** [review]

**Pull request merged**